scripts/mongodb-setup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo starting mongodb container on :27017
+
+docker run --name mongo -p 27017:27017 -d mongodb/mongodb-community-server:latest
```

Comment:
Better using an env variable on the host port
```suggestion
docker run --name mongo -p 27017:${MONGO_HOST_PORT:-27017} -d mongodb/mongodb-community-server:latest
```

Reply:
I think you meant it as
`docker run --name mongo -p ${MONGO_HOST_PORT:-27017}:27017 -d mongodb/mongodb-community-server:latest`
either way ok!